package.json

```diff
@@ -19,6 +19,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-map-gl": "^7.0.19",
+    "react-modal": "^3.16.1",
```

Comment:
The material ui library we use already comes with a modal component - I'd suggest using that, its probably fairly quick to switch between them!

https://mui.com/material-ui/react-modal/